modules/dqt/jsx/react.paginator.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+const PaginationLinks = (props) => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
don't disable eslint

Reply:
@driusan I'm unaware of any other way to resolve this eslint conflict unless disabling the eslint rule. The way function components use state will result in an unused var but even when it's being used.

Reply:
There is no conflict. If eslint complains that the variable for the return value of useState is used it means you're not using the state.

Reply:
@driusan I'll take a look because maybe not for this file but other file resulted in the code breaking if I removed the line where the state with the setter was set; where the eslint was complaining about. eslint was angry since either the setter or the actual state wasn't being used and ignoring that one of them was used.

Reply:
If the setter isn't used it means it's not state, it's a constant, and if the value isn't used it means it's not used.

Reply:
@driusan yah I think you're right. I'm trying to get another PR fixed and I'll try to fix all of these. Maybe the props value is all that's needed and I had a state for it but never used.

Reply:
@driusan latest commit fixes it.